news/2022-08-04-wiki-contributors-wanted.md

```diff
@@ -0,0 +1,25 @@
+---
+layout: post
+title: "Wiki contributors wanted!"
+date: 2022-08-04 10:00:00 +0000
+---
+
+Lore maestros! Technical aficionados! Information brokers! We seek any kind of knowledge to share in the wiki, and to translate it for every corner of the community.
+
+![](/wiki/shared/news/banners/osu-wiki.jpg)
+
+The [osu! wiki](/wiki/Main_Page) has seen a lot of reconstruction and development over the years. However, while the to-do list grows larger with new ideas and things to expand on, the amount of contributors has remained very small, which has impeded any rapid progress. We are therefore looking for more people to help out!
+
+If you're knowledgeable on certain aspects of the community, whether it be gameplay, mapping, modding, hitsounding, storyboarding or trivia, or simply want to exercise your writing skills, you're more than welcome to participate! No need for Ph.D.'s — any level of contribution is appreciated!
+
+The osu! community reaches every part of the world, which means translations are necessary to keep the knowledge base useful for everyone. By a quick look in the [list of outdated articles and missing translations](https://clayton.cc/osu-wiki-status/en), the wiki only has French and Indonesian translations almost completely up to date, while every other language has fallen far short over the years. Anyone is encouraged to take part in any amount!
```

Comment:
```suggestion
The osu! community reaches every part of the world, which means translations are necessary to keep the knowledge base useful for everyone. By a quick look in the [list of outdated articles and missing translations](https://clayton.cc/osu-wiki-status), the wiki only has French and Indonesian translations almost completely up to date, while every other language has fallen far short over the years. Anyone is encouraged to take part in any amount!
```
since this one doesn't seem particularly about the English page. it's just redirect for now but I might have a main page sometime